dev-doc/README.md

```diff
@@ -0,0 +1,68 @@
+chef-run/chef-apply is a tool to execute ad-hoc tasks on one or more target nodes using Chef Infra Client. To start with, familiarize yourself with chef-run’s arguments and flags by running chef-run -h
+link https://docs.chef.io/workstation/chef_run/
+
+## General development process :
+
+1. Fork this repo and clone it to your workstation.
+2. Create a feature branch for your change.
+3. Write code and tests.
+4. Push your feature branch to GitHub and open a pull request against master.
```

Comment:
we could restate as - "...and create a pull request against main branch of original repo.
the numbering could be set to 1., 1., 1.,.. the markdown rendering will automatically take care of incrementing.
'workstation' could be renamed as 'development system' to avoid confusion with Chef Workstation.